tslint.json

```diff
@@ -19,49 +18,6 @@
       "rxjs/Rx",
       "rxjs/internal/operators"
     ],
-    "import-group-ordering": {
```

Comment:
Why did you remove `"tslint-import-group-ordering"? `Was it causing issues? It's quite a good package and I really like the grouping of imports :(

Reply:
I removed it as part of #42 , as it's been flagged as potentially stale and no longer maintained. I think it's also not compatible with ES Lint which is what we'll be moving toward. But having said that, perhaps we could just look at removing or replacing `tslint-import-group-ordering` when we do the ES Lint migration, and leave it as is for now? I don't think it's causing any issues.

Reply:
I would personally prefer to do that. In MYA we were able to use another package once moved to eslint so this could be part of the migration from `tslint` to `eslint` (cc @thatguynamedandy).

Reply:
Sounds good, also great to hear you're already using a replacement package

Reply:
I've removed that commit for now, and we can revisit when we do the migration to ES Lint 👍

Reply:
works for me 👍